config/requests.yml.tmpl

```diff
@@ -6,6 +6,8 @@ defaults: &defaults
   proxy_base: http://library.princeton.edu/resolve/lookup?url=
   stackmap_base: https://library.princeton.edu/utils/map
   voyager_api_base: http://catalog.princeton.edu:7014
+  voyager_ub_id: 1@DB1
+  pulsearch_base: https://pulsearch.princeton.edu
```

Comment:
Is this really needed? I feel like the rails url/route helpers should be sufficient.